src/languages/php.md

```diff
@@ -140,6 +140,46 @@ This is the complete list of extensions that can be enabled:
 > see the up-to-date complete list of extensions after you SSH into
 > your environment. For PHP 7, use `ls /etc/php/*/mods-available`.
 
+## Custom PHP extensions
+
+It is possible to use an extension not listed here, but there are a few steps involved.
+
+1. Either download the extension .so file in your build hook or Git commit that to your Git repository.
+
+2. Provide a custom `php.ini` file in the application root (as a sibling of your `.platform.app.yaml` file) that loads the extension using an absolute path.  For example, if the extension is named `spiffy.so` and is in the root of your application, you would have a `php.ini` file that reads:
+
+```ini
+extension=/app/spiffy.so
+```
+
+3. Update your build flavor to none in your `.platform.app.yaml` file. You will need to provide a custom Composer command to use the extra extension. 
+
+```yaml
+build:
+    flavor: none
+```
+
+4. Add `PHP_INI_SCAN_DIR` environment variable and run `composer install` in your build hook.
```

Comment:
See comment on step 3. Also, `PHP_INI_SCAN_DIR` is not applicable to all extensions... It's only needed for ioncube...

We can actually remove step 3 and step 4 for general cases.